source/geometries/FullRingInfinity.cc

```diff
@@ -258,36 +294,37 @@ void FullRingInfinity::BuildCryostat()
       new G4Tubs("ACTIVE", inner_radius_, inner_radius_ + wide_active_depth,
                  axial_length_/2., 0, twopi);
     active_logic_ =
-      new G4LogicalVolume(active_solid, LXe, "ACTIVE");
+      new G4LogicalVolume(active_solid, LXe_, "ACTIVE");
     new G4PVPlacement(0, G4ThreeVector(0., 0., 0.), active_logic_,
                       "ACTIVE", LXe_logic_, false, 0, true);
 
     // Set the ACTIVE volume as an ionization sensitive det
-    IonizationSD *ionisd = new IonizationSD("/PETALO/ACTIVE");
+    IonizationSD* ionisd = new IonizationSD("/PETALO/ACTIVE");
     active_logic_->SetSensitiveDetector(ionisd);
     G4SDManager::GetSDMpointer()->AddNewDetector(ionisd);
 
     // Limit the step size in ACTIVE volume for better tracking precision
     active_logic_->SetUserLimits(new G4UserLimits(max_step_size_));
 
-    G4Material *kapton =
+    // Reflectant panels
+    G4Material* kapton =
         G4NistManager::Instance()->FindOrBuildMaterial("G4_KAPTON");
 
-    G4Tubs *kapton_int_solid =
+    G4Tubs* kapton_int_solid =
         new G4Tubs("KAPTON", inner_radius_ - kapton_thickn_, inner_radius_,
                     axial_length_ / 2., 0, twopi);
     G4LogicalVolume *kapton_int_logic =
         new G4LogicalVolume(kapton_int_solid, kapton, "KAPTON");
     new G4PVPlacement(0, G4ThreeVector(0., 0., 0.), kapton_int_logic,
-                      "KAPTON_INT", LXe_logic_, false, 0, true);
+                      "KAPTON_INT", LXe_logic_, false, 0, false);
 
-    G4Tubs* kapton_ext_solid =
-      new G4Tubs("KAPTON", inner_radius_ + wide_active_depth, inner_radius_ + wide_active_depth + kapton_thickn_,
-                 axial_length_/2., 0, twopi);
+    G4Tubs* kapton_ext_solid = new G4Tubs("KAPTON", inner_radius_ + wide_active_depth,
+    inner_radius_ + wide_active_depth + kapton_thickn_,
+                  axial_length_/2., 0, twopi);
```

Comment:
Why don't you join line 323 with 322? The length would be ok and it would reduce `G4Tubs` definition in 2 instead of 3.

Reply:
I've changed the indenting, I think it is read better now.